internal/handlers/handlers.go

```diff
@@ -1 +1,71 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func HandlerHTML(w http.ResponseWriter, r *http.Request) {
+	content, err := os.ReadFile("index.html")
+	if err != nil {
+		http.Error(w, "Ошибка чтения файла", http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.Write(content)
+}
+
+func Upload(w http.ResponseWriter, r *http.Request) {
+	err := r.ParseMultipartForm(32 << 20)
+	if err != nil {
+		http.Error(w, "Ошибка парсинга формы", http.StatusBadRequest)
+		return
+	}
+
+	file, header, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, "Ошибка получения файла", http.StatusBadRequest)
+		return
+	}
+
+	defer file.Close()
+
+	fileData, err := io.ReadAll(file)
+	if err != nil {
+		http.Error(w, "Ошибка чтения данных из файла", http.StatusInternalServerError)
```

Comment:
Это проблема клиента
```suggestion
		http.Error(w, "Ошибка чтения данных из файла", http.StatusBadRequest)
```